Userland/Libraries/LibWeb/Animations/Animation.cpp

```diff
@@ -622,7 +622,7 @@ Animation::Animation(JS::Realm& realm)
 void Animation::initialize(JS::Realm& realm)
 {
     Base::initialize(realm);
-    set_prototype(&Bindings::ensure_web_prototype<Bindings::AnimationPrototype>(realm, "Animation"));
+    set_prototype(&Bindings::ensure_web_prototype<Bindings::AnimationPrototype>(realm, "Animation"_fly_string));
```

Comment:
Just thinking out loud, this feels like it's screaming to be generated :thinking:

Reply:
Yup, definitely. After this, doing that is somewhere on my yak stack